api/libproject/devstart.rb

```diff
@@ -478,21 +478,30 @@ def run_gradle(cmd_name, args)
   :fn => ->(*args) { run_gradle("gradle", args) }
 })
 
+def connect_to_db(*args)
+  if args.empty?
+    args = "--database=workbench"
+  end
 
-def connect_to_db()
-  common = Common.new
+  op = WbOptionsParser.new("connect-to-db", *args)
+  op.add_option(
+      "--database [database]",
```

Comment:
Is this better than just typing `use cdr;` after getting into the SQL prompt?

Reply:
Maybe not.  Think I'll just drop this idea.